### PR TITLE
Devops test automation strategy for a successful CI/CD adoption

### LINKED ISSUE #4722: [DevOps] DevOps test automation strategy for a successful CI/CD adoption
#### Proposed title of the article
DevOps test automation strategy for a successful CI/CD adoption
#### Proposed article introduction
The foundation of DevOps culture is test automation. The outbreak of the COVID-19 pandemic has compelled several institutions to adopt automated testing throughout the SDLC for CI/CD, even if they had previously recognized its importance and necessity. Work from home (WFH) reality necessitates enhanced test automation, which makes implementation more difficult. Many hats are worn by IT developers, operations engineers, and testers, all of whom work in independent operating units as distinct organizations. Going forward, the key question is whether QA can improve and speed up product delivery by integrating the many clusters under the DevOps banner. And if such is the case, how might automated testing scenarios be implemented? We'll answer all of these issues and show you how to implement a solid testing automation plan in this article.

#### Key takeaways 
1. Project’s scope of automation
2. Appropriate testing automation tool
3.  Setting up a cloud-based test environment
4. Test case risk assessment
5. Run and manage test cases
6. Analyze continuously and remodel the strategy for improvement
#### Article quality
Automated testing is the first move in the right direction when you embark on your automation journey and due to that, this article will be one of a kind as it will explain the concept of automation testing and it will be straightforward for the novices to grip the content in it. In contrast with the almost similar content on the internet, this article will be in the context and the examples that will be offered in this article will be basic to attract both intermediates and beginners in automation testing
#### References
N/A

### PR DESCRIPTION
**Pull request by Kama-144**
This fixes #4722

### CODE REVIEW

**Comment by ahmadmardeni1** [comment]
Plagiarism check done ✅

**Comment by Kama-144** [comment]
@BrianaNzivu hello, are there any updates on this PR

**Comment by BrianaNzivu** [comment]
> @BrianaNzivu hello, are there any updates on this PR

Working on it ASAP.

**Review by BrianaNzivu — APPROVED** [review]
Hey @Kama-144    👋 .

First of all, welcome to Section - EngEd. :tada: 
We are happy to have you here.

Very good job on the article. It is a very interesting read and has minimal mistakes. This is very impressive considering you are a first-time contributor 👏 .

Just a few things I'd like to point out:
- I suggest that you check out our resources page which has links to various resourceful tools for writing just to minimize the changes being made by the reviewer.
- Lastly please go through ALL the corrections I have made so that you can not repeat the same mistakes.

That's it! :smile: 

A final review will be done now by a content moderator.
Otherwise, congratulations on publishing your first article🍾.

In case of any question or clarification (currently or in the future), do not hesitate to reach out via [Slack](https://app.slack.com/client/T5X14SH7G/D01A6MT673R) or my personal email (briananzivu@gmail.com). 
I am **ALWAYS** here to help.

It has been a pleasure reading and reviewing this article.

Looking forward to seeing you contribute more to EngEd. 🚀

Once again good job! 👍 

Kind Regards.

**Comment by Kama-144** [comment]
@BrianaNzivu Thanks

File: content/articles/devops-test-automation-strategy-for-a-successful-ci-cd-adoption/index.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+
+Some of the severe real-world instances of the biggest technology failures (which occurred years ago, when the world was not as technologically sophisticated) indicate what can happen when adequate testing is not done on respective systems/software. For instance, on April 26, 1994, an Airbus A300B4-622R was finishing its usual flight routine when right before its landing at Nagoya Airport, it crashed, inflicting a heavy casualty count of 264 people. 
+
+To date, the accident remains one of the most lethal mishaps in the history of China Airlines. Similarly, on August 14, 2003, after 2:00 PM, a high-voltage power line in northern Ohio brushed against some overgrown trees. It shut down completely, forcing 50 billion people to live without power for two days. The alarm system failure caused by software issues made it one of the most significant blackouts in North American history.
```
Comment (WanjaMIKE) [inline content/articles/devops-test-automation-strategy-for-a-successful-ci-cd-adoption/index.md:44]: 50 billion people. Are you sure ? @Kama-144

Comment (Kama-144) [reply content/articles/devops-test-automation-strategy-for-a-successful-ci-cd-adoption/index.md]: @WanjaMIKE hello it was a typo its 50m

**Comment by WanjaMIKE** [comment]
Merged #5027 into master

**Pull request merged**